### PR TITLE
chore(ecs): update API comments for ecs service

### PR DESCRIPTION
**Pull request by ShiChangkuo**
**What this PR does / why we need it**:

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note

```

## PR Checklist

* [ ] Tests added/passed.
* [ ] Documentation updated.
* [ ] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST='./huaweicloud' TESTARGS='-run=TestAccSomethingV0_basic'
...
=== RUN   TestAccSomethingV0_basic
--- PASS: TestAccSomethingV0_basic (70.75s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud       70.796s
```

### CODE REVIEW

File: huaweicloud/services/ecs/resource_huaweicloud_compute_instance.go [hunk 1]
```diff
@@ -41,22 +41,23 @@ var (
...
+// @API ECS DELETE /v1/{project_id}/cloudservers/{server_id}/metadata/{key}
+// @API ECS POST /v1.1/{project_id}/cloudservers/{server_id}/resize
+// @API ECS PUT /v1/{project_id}/cloudservers/{server_id}/os-reset-password
+// @API ECS POST /v1/{project_id}/cloudservers/{server_id}/tags/action
+// @API ECS POST /v2.1/{project_id}/servers/{server_id}/action
+// @API ECS GET /v1/{project_id}/cloudservers/{server_id}
+// @API ECS GET /v1/{project_id}/cloudservers/{server_id}/block_device/{volume_id}
+// @API ECS GET /v1/{project_id}/jobs/{job_id}
 // @API IMS GET /v2/cloudimages
+// @API EVS POST /v2.1/{project_id}/cloudvolumes/{id}/action
```
Comment (Lance52259) [inline huaweicloud/services/ecs/resource_huaweicloud_compute_instance.go]:
```suggestion
// @API EVS POST /v2.1/{project_id}/cloudvolumes/{volume_id}/action
```

File: huaweicloud/services/ecs/resource_huaweicloud_compute_servergroup.go [hunk 2]
```diff
@@ -16,11 +16,11 @@ import (
 	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
 )
 
-// @API ECS GET /v1/{project_id}/cloudservers/{serverID}
+// @API ECS POST /v1/{project_id}/cloudservers/os-server-groups
 // @API ECS POST /v1/{project_id}/cloudservers/os-server-groups/{id}/action
 // @API ECS GET /v1/{project_id}/cloudservers/os-server-groups/{id}
 // @API ECS DELETE /v1/{project_id}/cloudservers/os-server-groups/{id}
-// @API ECS POST /v1/{project_id}/cloudservers/os-server-groups
+// @API ECS GET /v1/{project_id}/cloudservers/{serverID}
```
Comment (Lance52259) [inline huaweicloud/services/ecs/resource_huaweicloud_compute_servergroup.go]:
```suggestion
// @API ECS GET /v1/{project_id}/cloudservers/{server_id}
```

Comment (ShiChangkuo) [reply huaweicloud/services/ecs/resource_huaweicloud_compute_servergroup.go]: fixed

**Pull request merged**